enterprise-suite/es-monitor-api/bootstrap-monitors.json

```diff
@@ -1,639 +1,738 @@
+        }
+      },
+      "node_memory_pressure": {
+        "monitorVersion": "1",
+        "model": "threshold",
+        "parameters": {
+          "metric": "node_memory_usable_percent",
+          "window": "5m",
+          "confidence": "1",
+          "severity": {
+            "warning": {
+              "comparator": "<",
+              "threshold": "5"
+            }
+          },
+          "summary": "memory pressure",
+          "description": "{{$labels.instance}} has low usable memory"
+        }
+      },
+      "node_disk_filling": {
+        "monitorVersion": "1",
+        "model": "predict",
+        "parameters": {
+          "metric": "node_filesystem_free_percent",
+          "period": "1h",
+          "comparator": "<",
+          "threshold": "0",
+          "window": "15m",
+          "confidence": "1",
+          "severity": {
+            "warning": {
+              "seconds": "28800"
+            }
+          },
+          "summary": "disk filling",
+          "description": "{{$labels.instance}} will fill disk in {{$labels.prediction}}"
+        }
+      },
+      "node_network_errors": {
+        "monitorVersion": "1",
+        "model": "threshold",
+        "parameters": {
+          "metric": "node_network_error_rate",
+          "window": "5m",
+          "confidence": "1",
+          "severity": {
+            "warning": {
+              "comparator": ">",
+              "threshold": "0"
+            }
+          },
+          "summary": "network errors",
+          "description": "{{$labels.instance}} has network errors"
+        }
+      },
+      "kube_node_pressure": {
+        "monitorVersion": "1",
+        "model": "threshold",
+        "parameters": {
+          "metric": "kube_node_pressure",
+          "window": "5m",
+          "confidence": "1",
+          "severity": {
+            "warning": {
+              "comparator": ">",
+              "threshold": "0"
+            }
+          },
+          "summary": "kube node pressure",
+          "description": "{{$labels.node_name}} is in kubernetes condition {{$labels.condition}}"
+        }
+      },
+      "kube_container_restarts": {
+        "monitorVersion": "1",
+        "model": "threshold",
+        "parameters": {
+          "metric": "kube_pod_container_restarts_rate",
+          "window": "5m",
+          "confidence": "1",
+          "severity": {
+            "warning": {
+              "comparator": ">",
+              "threshold": "0"
+            }
+          },
+          "summary": "container restarting rapidly",
+          "description": "container {{$labels.container}} in pod {{$labels.pod}} of {{$labels.es_workload}} restarting rapidly"
+        }
+      },
+      "kube_pod_not_ready": {
+        "monitorVersion": "1",
+        "model": "threshold",
+        "parameters": {
+          "metric": "kube_pod_ready",
+          "window": "5m",
+          "confidence": "1",
```

Comment:
confidence changed from 0.5 to 1

Reply:
look through the "dial fail monitors to 100% confidence" change in the commit log to understand that one.